lib/optimist.rb

```diff
@@ -84,7 +84,7 @@ def self.registry_getopttype(type)
   ##  ignore options that it does not recognize.
   attr_accessor :ignore_invalid_options
 
-  DEFAULT_SETTINGS = { suggestions: true }
+  DEFAULT_SETTINGS = { suggestions: true, exact_match: false }
```

Comment:
I think this should be defaulted to true.

```suggestion
  DEFAULT_SETTINGS = { suggestions: true, exact_match: true }
```